packages/core/src/types/oas3.ts

```diff
@@ -300,6 +300,15 @@ const Schema: NodeType = {
         return 'Schema';
       }
     },
+    additionalItems: (value: any) => {
+      if (typeof value === 'boolean') {
+        return { type: 'boolean' };
+      } else if (Array.isArray(value)) {
+        return listOf('Schema');
+      } else {
+        return 'Schema';
```

Comment:
Items could be of type schema? I see [this](https://github.com/Redocly/redocly-cli/pull/707/files#diff-d9fcca14c2dc15ba8f97e02b116086a816b6c45f77b7f2d6451de2c190490f0eR300), but curious how does it work.

Reply:
You can see it in [spec](https://json-schema.org/understanding-json-schema/reference/array.html#additional-items).
and check how it works on redoc [PR](https://github.com/Redocly/redoc/pull/2018).

```yaml
items:
 type: object
...
additionalItems:
  type: string
```